COMMERCE-11737

victorcdp opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

katienesterovich commented:
ci:test:sf

katienesterovich commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-9.liferay.com/job/test-portal-source-format/965/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-9.liferay.com//userContent/jobs/test-portal-source-format/builds/965/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/5400/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-14.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5400/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 38 seconds 585 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0418b5f0674ba6fc98653a165fea5d30b5408795">0418b5f0674ba6fc98653a165fea5d30b5408795</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-107295">LPS-107295</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/028ad26b7f3094ef3d5bf7021cd66d6f4bb22721">028ad26b7f3094ef3d5bf7021cd66d6f4bb22721</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-9.liferay.com/job/test-portal-source-format/965/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/965/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:x: ci:test:relevant - 24 out of 26 jobs passed in 1 hour 23 minutes 7 seconds 268 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fd8ffae97e66755c1ab4f09f0e54ae4b3a6046dc">fd8ffae97e66755c1ab4f09f0e54ae4b3a6046dc</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b48e22ed1129b772ad8bdbb965fb3eccc584f484">b48e22ed1129b772ad8bdbb965fb3eccc584f484</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221592/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/315540/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352684/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221590/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353246/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/355491/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352686/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345625/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286980/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 24 out of 26 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/5400/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221591/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286979/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353247/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221592/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345626/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/355492/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/315541/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/300217/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/354283/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/315540/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352684/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221590/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353246/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/355491/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313901/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313900/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/315539/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/354284/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352686/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/355490/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345625/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313902/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/300216/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352685/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/286980/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5400/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at fd8ffae97e66755c1ab4f09f0e54ae4b3a6046dc:</strong></summary><ol><li><div><h5><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221591/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></h5><div><h6>Job Results:</h6><p>210 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/221591//consoleText">AXIS_VARIABLE=0,label_exp=!master #221591</a><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/221591//testReport/com.liferay.project.templates/ProjectTemplateFilesTest/testProjectTemplateFiles">ProjectTemplateFilesTest.testProjectTemplateFiles</a><pre><code>org.junit.ComparisonFailure: Forbidden leading or trailing whitespaces in ../project-templates-npm-angular-portlet/src/main/resources/archetype-resources/.babelrc.js expected:&lt;module.exports = {};[]&gt; but was:&lt;module.exports = {};[
]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at com.liferay.project.templates.ProjectTemplateFilesTest._testTextFile(ProjectTemplateFilesTest.java:926)
	at com.liferay.project.templates.ProjectTemplateFilesTest._testTextFile(ProjectTemplateFilesTest.java:951)
	at com.liferay.project.templates.ProjectTemplateFilesTest.access$400(ProjectTemplateFilesTest.java:70)
	at com.liferay.project.templates.ProjectTemplateFilesTest$1.visitFile(ProjectTemplateFilesTest.java:895)
	at com.liferay.project.templates.ProjectTemplateFilesTest$1.visitFile(ProjectTemplateFilesTest.java:840)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.project.templates.ProjectTemplateFilesTest._testProjectTemplateFiles(ProjectTemplateFilesTest.java:838)
	at com.liferay.project.templates.ProjectTemplateFilesTest.testProjectTemplateFiles(ProjectTemplateFilesTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1...</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/720

Kvale1733 committed:
LRQA-79973 Add properties to run locale tests in language module

JerryAdrianoo committed:
LPS-182074 Add ClientExtensionRendererCell testcase

amosfong committed:
COMMERCE-11005 fix upsert if channel doesn't exist

thektan committed:
LPS-182969 learn-api: Add getReactDataJSONObject method

thektan committed:
LPS-182969 frontend-js-components-web: Add LearnMessage component

thektan committed:
LPS-182969 frontend-js-components-web: Generate types

ealonso committed:
LPS-186529 Update compatibility layer to have only one method, since it is used in only one place in portal

ealonso committed:
LPS-186529 Uses new method

ealonso committed:
LPS-186529 Create map only if it is necessary

ealonso committed:
LPS-186529 Update package

ealonso committed:
LPS-186529 Moves to portlet display template api module

ealonso committed:
LPS-186529 Uses api classes

ealonso committed:
LPS-186529 Uses util classes directly

ealonso committed:
LPS-186529 baseline

ealonso committed:
LPS-186529 build.gradle

ealonso committed:
LPS-186529 Update tests

BarbaraCabrera committed:
LPS-186419 Replace the control-menu-icon where we can use monospaced and small button classes

(cherry picked from commit 0ab2630fb9a063916e52be5b4d6ff833c621d9fe)

JerryAdrianoo committed:
LPS-182074 Creating paths to collapse icon and field names

JerryAdrianoo committed:
LPS-182074 Add openOrCloseCollapseField macro

thektan committed:
LPS-182969 learn-api: gradle baseline

joelrocha68 committed:
LPS-186318 Remove quarantine for CanEditAnswer test

joelrocha68 committed:
LPS-186318 Add editComment macro

joelrocha68 committed:
LPS-186318 Add path to questions details view

joelrocha68 committed:
LPS-186318 Add CanEditComment test

joelrocha68 committed:
LPS-186318 SF

BarbaraCabrera committed:
LPS-186419 Replace the use of control-menu-icon where we can use monospaced and small button classes

(cherry picked from commit c80f246ee9e82d4751c063cf0869f719749cb560)

BarbaraCabrera committed:
LPS-186419 Use Clay:button instead of Liferay-ui:icon and replace the use of control-menu-icon where we can use monospaced and small classes

(cherry picked from commit 89c785db58de0e0e46d4282b21a9093ae5c8daed)

BarbaraCabrera committed:
LPS-186419 Replace the use of control-menu-icon because the element no longer uses it but it uses lfr-back-link

(cherry picked from commit eab04343104c0af234bba3e05879f9976c63187d)

BarbaraCabrera committed:
LPS-186419 Replace the use of control-menu-icon with monospaced and small button classes when using the back button

(cherry picked from commit fcdaf763d5b42661528882e02af2fbbea744dd91)

BarbaraCabrera committed:
LPS-186419 Remove not needed control-menu-icon classes

(cherry picked from commit 545205e433f8b5941cc435a11868c9eae4bf125f)

ruben-pulido committed:
LPS-185604 Return empty JSON if action execution was successful or JSON with error field if action execution failed

Stephy6 committed:
LPS-183306 Clean ManageCollaborators.scss

Stephy6 committed:
LPS-183306 Clean TreeFilter.scss

Stephy6 committed:
LPS-183306 Clean _VocabulariesSelection.scss

Stephy6 committed:
LPS-183306 Use clay guidelines for DownloadSpreadsheetButton components

Stephy6 committed:
LPS-183306 Use clay guidelines for AuditGraphApp components

Stephy6 committed:
LPS-183306 Use clay utilities and guidelines from .jsp files

BarbaraCabrera committed:
LPS-186419 Remove no needed control-menu-icon and padding

Also changing the margin to follow the established patterns

(cherry picked from commit e643dac30510b61fac83dc4aef519b888e9a33e7)

BarbaraCabrera committed:
LPS-186419 Delete unused css class control-menu-icon

(cherry picked from commit d0dff045fe3e605d784cd831b489d199109bda12)

claraizquierdo committed:
LPS-186629 Change the styles of the width and height inputs

Stephy6 committed:
LPS-183306 Remove unnecessary loading-animation

claraizquierdo committed:
LPS-186629 Add Feature Flag

Commit:
LPS-186327 Create UpgradeJavaReferenceAnnotationCheck

Commit:
LPS-186327 Create test UpgradeJavaReferenceAnnotationCheck

JerryAdrianoo committed:
LPS-182074 Add CanCreateCellRendererViaOSGI Test

ruben-pulido committed:
LPS-185604 Add method returning component context with URL for executing an action

Kvale1733 committed:
LRQA-79379 Remove sikuli assert

thektan committed:
LPS-183502 LPS-186653 frontend-data-set-views-web: Default sort by create date

thektan committed:
LPS-183502 LPS-186653 frontend-data-set-views-web: Allow columns to be sorted

shuyangzhou committed:
LPS-187401 Initial checkin

shuyangzhou committed:
LPS-187401 Bump version

shuyangzhou committed:
LPS-187401 Apply usages

shuyangzhou committed:
LPS-187401 Patch NullPackageSource for better concurrency.

jkappler committed:
LPS-186372 Adds test for page definition with a fragment with multiple translations. Note that the definition when creating is a JSONObject, but when reading, it's a String, so that's why it's handled in differently

jkappler committed:
Revert "LPS-186285 SF"

This reverts commit 75d24b4852cd46a71ea0b1581c68a5c7fbfe1176.

jkappler committed:
Revert "LPS-186285 Use importer to import the page definition directly in the structure if it's available, this will take also care of the other properties that we moved previously"

This reverts commit 94782654b9fd79bb34afc1b87f453e1d859f432b.

jkappler committed:
Revert "LPS-186285 Let the importer handle the elements from page definition, such as favicon and client extensions"

This reverts commit bb31eae47582a3ffc4541aacf10017e7d4d894fe.

jkappler committed:
LPS-186285 Use importer to import the page definition directly in the structure if it's available

jkappler committed:
LPS-186372 Adds tests for page definition settings that have either a favicon from either a file entry or a client extension

jkappler committed:
LPS-186372 Adds tests for page definition with valid css and js client extensions

jkappler committed:
LPS-186372 Adds tests for page definition with a client extension of invalid type

jkappler committed:
LPS-186372 Adds tests for site navigation menu settings

jkappler committed:
LPS-186372 Adds tests for custom fields

Commit:
LPS-186327 Generated Documentation

thektan committed:
LPS-182969 frontend-js-components-web: Add ILearnResourceContext interface

thektan committed:
LPS-182969 frontend-js-components-web: Allow anchor element props

ngriffin7a committed:
LPS-186990 Define segments based on User and Organization Categories

thektan committed:
LPS-182969 frontend-js-components-web: yarn types

Commit:
LPS-186001 Migrate to DSLQuery

Commit:
LPS-186001 Remove old finder

Commit:
LPS-186001 BuildService

sandrodw3 committed:
LPS-183306 Adapt spacing

sandrodw3 committed:
LPS-183306 Remove position rules

sandrodw3 committed:
LPS-183306 This CSS is not being applied

ruben-pulido committed:
LPS-185604 Set renderActionHandler to false when rendering LayoutStructure in a Drop Zone.

This avoids including the handler multiples times in the page, and prevents actions from being executed several times when a button is clicked.

ruben-pulido committed:
LPS-185604 Render InfoItemActionHandler Component

ruben-pulido committed:
LPS-185604 Add boolean parameter to control whether to render Action Handler

fmarcio committed:
LRAC-13975 replace Link component by ClayLink

fmarcio committed:
LRAC-13975 update snapshots

sandrodw3 committed:
LPS-185604 Add InfoItemActionHandler js component

sandrodw3 committed:
LPS-185888 Allow passing itemSelectorURL to MappingSelector

regiCesar21 committed:
LRAC-13964 change size to default

regiCesar21 committed:
LRAC-13964 test

thektan committed:
LPS-182969 frontend-js-components-web: Extend clay link props

jonathanmccann committed:
LPS-187080 These values are comma separated so split them prior to processing them

danielCamara02 committed:
LPS-185034 Add CanSavePublicationsSettingsWhenItIsDisabled test

Commit:
LPS-186419 Updade WorkflowMetricsSLA#METRICS_MENU_SLA path

(cherry picked from commit 40c901e537a33f9d60c6e11db0b789d747cfef63)

thektan committed:
LPS-182969 frontend-js-components-web: Prevent undefined errors

thektan committed:
LPS-182969 frontend-js-components-web: Add jest test for LearnMessage

thektan committed:
LPS-182969 frontend-js-components-web: Add helpful console warnings

vicnate5 committed:
LRQA-80929 Reset database should reset schema instead of container

shuyangzhou committed:
LPS-187404 Don't register CounterTransactionExecutor as osgi service.

shuyangzhou committed:
LPS-187404 Simplify AopService and TransactionExecutor binding process

jkappler committed:
LPS-186372 Hide tests behind FF

sandrodw3 committed:
LPS-185888 Use new url to show info items with actions

sandrodw3 committed:
LPS-185604 baseline

IstvanD committed:
LPS-180849 Only check for update permission on the published layout if the page is being edited

Commit:
LPS-186862 Delete interestTerms in order to remove contentRecommendation

jorgediaz-lr committed:
LPS-168539 Apply BundleUtil.getBundle usage

Commit:
LPS-186862 Delete processor in order to remove ContentRecommendation

Commit:
LPS-186862 Delete processor tests

Commit:
LPS-186862 Remove contentRecommendation related labels

Commit:
LPS-186562 buildLang

JerryAdrianoo committed:
LPS-182074 Add CanHiddenIdentityField test

JerryAdrianoo committed:
LPS-182074 Add CanChangeLanguage test

JerryAdrianoo committed:
LPS-182074 Add CannotPublishWhenFieldNameIsEmpty test

JerryAdrianoo committed:
LPS-182074 Add CanHiddenContentField test

Commit:
LPS-186862 Remove assetListAssetEntryQueryProcessor usages

Commit:
LPS-187164 Remove groovy script from payment confirmation

marcosapmf committed:
LRAC-13967 add button attribute

marcosapmf committed:
LRAC-13967 Regen snapshots

JerryAdrianoo committed:
LPS 182074 Add CannotPublishWhenJSFieldIsEmpty test

JerryAdrianoo committed:
LPS-182074 Add CanHiddenAdditionalResourcesField test

JerryAdrianoo committed:
LPS-182074 Add CannotPublishWithInvalidValueOnSourceCodeField test

JerryAdrianoo committed:
LPS-182074 Add CanCreateCellRender test

JerryAdrianoo committed:
LPS-182074 Add CanCancelChanges test

JLucasM committed:
LRAC-13945 Add a display-card and loading-root class in components

JLucasM committed:
LRAC-13945 Update Snapshots

Commit:
LRAC-13926 Update Clay Picker version

Commit:
LRAC-13926 - Update snapshot

itstali committed:
LPS-154598 Create AssertIncompleteD2CApplicationAppearsInAP test

kevhlee committed:
LPS-186327 Use JavaSourceUtil.getAnnotationBlocks() instead to check annotations

kevhlee committed:
LPS-186327 Auto-SF

vicnate5 committed:
LRQA-80929 Reset container when installing new type of bundle to do full setup and correct portal-ext

shuyangzhou committed:
LPS-187404 Fix test

jkappler committed:
LPS-186372 Correct namings

Hijky committed:
LPS-177402 Update a macro to get siteId

Hijky committed:
LPS-177402 Update a testcase to add cases for manage keywords associated to object entries

ealonso committed:
LPS-186151 Dont do certain actions if we are in a delete site process

Commit:
LPS-186001 Baseline

BarbaraCabrera committed:
LPS-186419 Fix test due to changes in HeaderBackButton.es.js

Commit:
LPS-186682 Remove export processor package

Commit:
LPS-186682 Baseline

ealonso committed:
LPS-186529 build.gradle

Commit:
LPS-183502 ignore FDSEntry#CanChangePaginationInAdminPage util LPS-187299 solved

Commit:
LRAC-14015 Change contact sales button display type

Commit:
LRAC-14015 Update snapshots

ling-alan-huang committed:
LPS-183212 SF

victorg1991 committed:
LPS-185604 Instead of calling doTag which set a bad servletContext execute lifecycle event manually

sandrodw3 committed:
LPS-185604 Move logic to separated function and use it to render pagination component as well

ealonso committed:
LPS-186990 Adds language key

ealonso committed:
LPS-186990 buildLang

ealonso committed:
LPS-186990 Removes no needed files

ealonso committed:
LPS-186990 Adapt logic to take into account the category selection

ealonso committed:
LPS-186990 Get vocabularies only for the global site, since in user and organization edition we can only use global site categories

rafaellajordao committed:
LPS-178990 bug automation test ViewFragmentsAfterDiscardDraft

ealonso committed:
LPS-186990 Simplify using ListUtil

ealonso committed:
LPS-186990 Disabled it for the moment

Stephy6 committed:
LPS-183074 Remove default properties from page definition in display-page-templates folder

Stephy6 committed:
LPS-183074 Remove default properties from page definition in master-pages folder

Stephy6 committed:
LPS-183074 Remove default properties from page definition in layout-utility-page-entries folder

Stephy6 committed:
LPS-183074 Remove default properties from page definition in layout folder

itstali committed:
LPS-154598 SF

pei-jung committed:
LPS-186455 Semver

Commit:
LPS-178990 Sort

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b39cfc02f86bcd5f711b997e5ffd2c72ad30764b">b39cfc02f86bcd5f711b997e5ffd2c72ad30764b</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-11737">COMMERCE-11737</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/5883aecf29f95ab60dbdb8a1e43a865604d7c11a">5883aecf29f95ab60dbdb8a1e43a865604d7c11a</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-38.liferay.com/job/test-portal-source-format/4642/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/4642/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-source-format/4642/">test-portal-source-format#4642</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/4642/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/91">brianikim#91</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2348375411">[master] ci:test:sf - victorcdp &gt; brianikim - PR#91 - 2023-06-12[15:03:50]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/16705/">publish-testray-report#16705</a><br/></div></div>

xbrianlee committed:
LPS-184613 Do not run test on release jobs due to particular timing

brianikim commented on modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartResourceImpl.java:
```diff
@@ -336,9 +336,12 @@ private CommerceOrder _addCommerceOrder(
 			commerceCurrencyId, _getCommerceOrderTypeId(cart));
+
+		return _commerceOrderService.updatePrintedNote(
+			commerceOrder.getCommerceOrderId(), cart.getPrintedNote());
```
This is the wrong place to do it. When the order is being added, only the ADD_TO_CART permission needs to be considered. PrintedNote update needs to be done as part of `_updateOrder`

victorcdp replied:
@brianikim Is it alright now?

CalumR23 committed:
POSHI-558 Add check if selenium method is grabbed

CalumR23 committed:
POSHI-558 Update com.liferay.poshi.core to 1.0.120

Commit:
LPS-181529 Remove set up virtual instance and tear down block from Navigation

Commit:
LPS-181529 Add locator for item entry add icon

Commit:
LPS-181529 Modify locator to adapt new markup

Commit:
LPS-181529 Add macro to add sibling item

Commit:
LPS-181529 Add case to add sibling navigation menu item

Commit:
LPS-181529 Add macro to add child item

Commit:
LPS-181529 Add case to add child navigation menu item

Commit:
LPS-181529 Update coverage info

Commit:
LPS-181529 Add case to open contextual menu via View Info action

brianchandotcom committed:
COMMERCE-11005 SF

brianchandotcom committed:
POSHI-558 prep next

Commit:
LPS-181529 Refactor locator

Commit:
LPS-181529 Refactor macros

Commit:
LPS-181529 Add case to add sibling navigation menu items via keyboard

brianchandotcom committed:
LPS-185604 SF

brianchandotcom committed:
LPS-187404 Wordsmith

brianchandotcom committed:
LPS-187401 prep next

Commit:
LPS-181529 Add case to add child navigation menu item via keyboard

Commit:
LPS-181529 Add steps to assert screen reader announce relevant info

brianchandotcom committed:
LPS-186327 prep next

Commit:
LPS-187451 Use random name to avoid conflict

brianchandotcom committed:
LPS-186327 prep next

Commit:
COMMERCE-11737 Add Printed Note when creating Order through PostCart

brianikim commented on modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartResourceImpl.java:
```diff
@@ -755,6 +755,9 @@ private void _updateOrder(CommerceOrder commerceOrder, Cart cart)
 
+		_commerceOrderService.updatePrintedNote(
+			commerceOrder.getCommerceOrderId(), cart.getPrintedNote());
+
```
Still not correct.
```
			commerceOrder = _commerceOrderService.updatePrintedNote(
				commerceOrder.getCommerceOrderId(),
				GetterUtil.getString(
					order.getPrintedNote(), commerceOrder.getPrintedNote()));
```

Once you made this change, you can send it to liferay-commerce and assign it to me there.

victorcdp replied:
@brianikim In the GetterUtil, it should be `cart.getPrintedNote()` instead of `order.getPrintedNote()` right? Also, why assign it to a variable if it isn't gonna be used? just Liferay convention?

brianikim replied:
Sorry, yes `cart.getPrintedNote()` is correct. Also, what do you mean it isn't used? The updated commerceOrder will be used in line 768 and 773.

victorcdp replied:
oh i hadn't seen that. Alright, i'll make the changes. Thank you!
